coast/TIDEGAUGE.py

```diff
@@ -767,34 +754,35 @@ def read_bodc_header(fn_bodc):
         -------
         dictionary of attributes
         """
-        debug(f'Reading BODC header from "{fn_bodc}"')
+        debug(f"Reading BODC header from \"{fn_bodc}\"")
         fid = open(fn_bodc)
```

Comment:
Another place to use Path.read_text or a context manager.

Reply:
noted for future optimization requirement, if performance takes a hit.